pootle/apps/pootle_app/views/admin/projects.py

```diff
@@ -41,6 +41,7 @@ def get_context_data(self, **kwargs):
                 'checkstyle': Project.checker_choices,
                 'filetypes': filetypes,
                 'source_language': language_choices,
+                'template_tp': language_choices,
```

Comment:
But `language_choices` is a `Language` queryset, not a TP queryset. But even if we want to use languages, `templates` language is explicitly being excluded.